manifests/f/FireseedNetwork/Theseus2000/1.2/FireseedNetwork.Theseus2000.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: FireseedNetwork.Theseus2000
+PackageVersion: 1.2
+PackageLocale: en-US
+Publisher: Fireseed Network
+PackageName: Theseus2000
+License: Freeware
+ShortDescription: Theseus2000 Setup

```

Guess you missed this one from the old PR
```suggestion
ShortDescription: The high powered networking suite for PCs!
```